project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 80% AFUE Propane, 5F-40F switchover band
+        apply_logic: *dual-fuel_80pctAFUE_propane_furnace_apply_logic     
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_92-5_propane
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 92.5% AFUE Propane, 5F-40F switchover band
+        apply_logic: *dual-fuel_92-5pctAFUE_propane_furnace_apply_logic   
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_60_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 60% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: *dual-fuel_60pctAFUE_FO_furnace_apply_logic   
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_76_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 76% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: *dual-fuel_76pctAFUE_FO_furnace_apply_logic     
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_80_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 80% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: *dual-fuel_80pctAFUE_FO_furnace_apply_logic     
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+      - &ducted_ASHP_HigherEfficiency_exist_backup_shared_ducts_92-5_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 20, 11 HSPF, CCHP, Integrated Backup, 92.5% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+        costs: *costs_ashp_ducted_SEER_20_11_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_20_11_HSPF
+
+  - upgrade_name: EnergyStar ductless MSHP w/ exist backup  #also provides SEER 15 cooling measure profile
+    # Measure 4
+    options:
+      # MSHP
+      - &ductless_MSHP_EnergyStar_exist_backup
+        option: HVAC Heating Efficiency|MSHP, SEER 16, 9.2 HSPF, Separate Backup # Changed to Max Load
+        apply_logic: &ductless_MSHP_elec_backup_apply_logic
+          - and:
+            - HVAC Has Ducts|No
+            - or:
+              - not: HVAC Cooling Efficiency|Non-Ducted Heat Pump
+              - not: HVAC Cooling Efficiency|Ducted Heat Pump
+              - HVAC Heating Efficiency|MSHP, SEER 14.5, 8.2 HSPF
```

Comment:
missing exclusion of shared HVAC (should be upgrade-global, don't think it's here at all)